core/scripts/testing/styles.css

```diff
@@ -32,19 +32,13 @@
 }
 
-:root, html {
-  --ion-dynamic-font: var(--ion-default-dynamic-font);
```

Comment:
This variable was removed in [v8](https://github.com/ionic-team/ionic-framework/blob/next/BREAKING_ARCHIVE/v8.md#dynamic-font).